packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1892,7 +1906,7 @@ function flushRoot(root: FiberRoot, expirationTime: ExpirationTime) {
   performWorkOnRoot(root, expirationTime, true);
   // Flush any sync work that was scheduled by lifecycles
   performSyncWork();
-  finishRendering();
```

Comment:
This was unnecessary, since `performWork` (called by `performSyncWork`) calls `finishRendering` at the end.